src/urllib3/connection.py

```diff
@@ -80,31 +107,59 @@ class HTTPConnection(_HTTPConnection):
       Or you may want to disable the defaults by passing an empty list (e.g., ``[]``).
     """
 
-    default_port = port_by_scheme["http"]
-
-    #: Disable Nagle's algorithm by default.
-    #: ``[(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)]``
-    default_socket_options = [(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)]
```

Comment:
`default_socket_options` shouldn't be removed

Reply:
Right, got confused about that. Now I see that it is shared across all instances.